README.md

```diff
@@ -72,6 +72,8 @@ A curated list of awesome research papers, datasets and software projects devote
 
 #### Program Synthesis and Induction
 
+* <img src="badges/12-pages-beginner-brightgreen.svg" alt="11-pages" align="top"> [Compiler Fuzzing through Deep Learning](https://chriscummins.cc/pub/2018-issta.pdf) - Chris Cummins, Pavlos Petoumenos, Alastair Murray, Hugh Leather, ISSTA 2018
+* <img src="badges/12-pages-beginner-brightgreen.svg" alt="12-pages" align="top"> [Synthesizing Benchmarks for Predictive Modeling](https://chriscummins.cc/pub/2017-cgo.pdf) - Chris Cummins, Pavlos Petoumenos, Zheng Wang, Hugh Leather, CGO 2017
```

Comment:
We sort our paper chronologically. If you could move this one a few lines down that'd be perfect! Also:
```suggestion
* <img src="badges/14-pages-beginner-brightgreen.svg" alt="14-pages" align="top"> [Synthesizing Benchmarks for Predictive Modeling](https://chriscummins.cc/pub/2017-cgo.pdf) - Chris Cummins, Pavlos Petoumenos, Zheng Wang, Hugh Leather, CGO 2017
```